src/pymovements/plotting/traceplot.py

```diff
@@ -108,6 +108,23 @@ def traceplot(
     x_signal = gaze.samples[position_column].list.get(0)
     y_signal = gaze.samples[position_column].list.get(1)
 
+    screen_width_px = None
+    screen_height_px = None
+
+    if gaze.experiment is not None:
+        screen = gaze.experiment.screen
+        screen_width_px = screen.width_px
+        screen_height_px = screen.height_px
+
+        if screen_width_px is not None and screen_height_px is not None:
```

Comment:
I am not sure if this is the right approach. It wouldn't let users specify the figure size anymore.

What was the original reasoning to put this in? To adjust the figure size depending on the screen aspect ratios? Then probably [`axis('equal')`](https://matplotlib.org/stable/gallery/subplots_axes_and_figures/axis_equal_demo.html) would be a better approach.

I'll write an issue for the aspect ratios.